kubernetes/services/signing-service.yml.j2

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind:       Deployment
+metadata:
+  name: signing-service
+spec:
+  replicas:             1
+  revisionHistoryLimit: 1
+  template:
+    metadata:
+      labels:
+        run: signing-service
+    spec:
+      containers:
+        - name:             signing-service
+          image:            {{ docker_registry }}/{{image_prefix }}signing-service:{{ image_version }}
+          imagePullPolicy:  Always
+          args:             [
+              "{{ cluster_address }}",
+              "5",
```

Comment:
`initial_reconnect_delay` has a default now so we can stop hard-coding it here: https://github.com/golemfactory/concent/issues/692

Reply:
Done.